src/compute/compute.py

```diff
@@ -481,7 +481,11 @@ def submit_job_path():
             header = {"X-Permission-Denied": "User does not have permissions to access machine or path"}
             return jsonify(description="Failed to submit job"), 404, header
 
-    targetPath = request.form["targetPath"]
+    try:
+        targetPath = request.form["targetPath"]
+    except Exception as e:
+        data = jsonify(description="Failed to submit job", error="'targetPath' parameter not set in request")
```

Comment:
why is this not?

data = jsonify(description="Failed to submit job file",error=e)

however, I cannot foresee any other error other than the once in L490, no?

Reply:
Since the try-except block is wrapping only L485's sentence, we considered that it is fair to state that the error is that the parameter is not set.

Reply:
just not to catch something unexpected what about using
"except KeyError"

Reply:
LGTM